src/msf/mod.rs

```diff
@@ -11,10 +11,22 @@ use source::*;
 mod page_list;
 use self::page_list::PageList;
 
+use scroll::{Pread, LE};
 use std::fmt;
 
 type PageNumber = u32;
 
+/// The PDB header as stored on disk.
+#[derive(Debug, Pread)]
+struct RawHeader {
```

Comment:
You may want this to be repr(C) or packed (I’m not sure how the disk format is, but I’ve noticed ms structs tend to be packed sometimes)

Reply:
Yeah, that seems correct:
https://github.com/luser/dump_syms/blob/772f6d35664e2cec57e0fd4d4fde87514a642d0e/PDBHeaders.h#L33

Does that actually matter for `scroll_derive` purposes, though? It looked like it just reads each field individually, right?

Reply:
Yes, unfortunately it can matter but I think it’s fine? It’s more important if you read sequences of structs so yea here shouldn’t matter

Reply:
Switching to `scroll_derive` would be a good opportunity to massage `pdb` structs towards their on-disk counterparts, especially in light of #16. `#[repr(C)]` mirroring the upstream `struct`s seems like the right direction.

Reply:
I'd just like to state for the record that I lost like 15 minutes trying to figure out why the [`szMagic`](https://github.com/Microsoft/microsoft-pdb/blob/082c5290e5aff028ae84e43affa8be717aa7af73/PDB/msf/msf.cpp#L948) field in the Microsoft code is only defined as 30 bytes long, but the definition in all the other implementations I've seen is 32 bytes. (Spoiler: the Microsoft code *does not* pack its structs, so those extra 2 bytes are padding!)